src/templates/users/userprofile_detail.html

```diff
@@ -34,6 +34,8 @@
                   <a href="{% url 'users:add-friend' object.user.pk %}"
                      class="btn btn-primary">Add Friend</a>
                 {% endif %}
+                <a href="{% url 'users:user-history' object.user.pk %}"
+                   class="btn btn-primary">Stats</a>
```

Comment:
Once you merge dev, you will see that the Edit Profile and Account buttons are sectioned off separately from the general profile buttons such as the Friend Request buttons and the Stats button. The Friend Request and stats buttons should be wrapped such that they either appear alongside the other buttons or such that there is space between the two rows of buttons.

Reply:
Got it, I'll add them into the if statements just like I did with the account button so it looks cleaner